blog/content/post/2020-03-24-trenchboot-nlnet-introduction.md

```diff
@@ -0,0 +1,107 @@
+1. Implement DRTM with TrenchBoot for AMD processors.
+2. Create test environment with test suites for users.
+
+Above issues will be described later. Both, as already mentioned, will be fully
+open-source and available for everybody. Solution, which we provide will be
+intended for AMD processors because it doesn't include any closed
+components.
+
+Now, let me briefly discuss each goal, so you can better understand the project.
+Of course, we won't focus on details and all requirements now. Those will be
+systematically updated and presented to community.
+
+#### DRTM with TrenchBoot
+
+TrenchBoot is one of the stages of boot process. It runs directly after firmware
+(coreboot in our case) execution ended. It is a technology which measures and
+verifies running environment. To obtain that, TrenchBoot utilizes **Dynamic Root
+of Trust for Measurements** (DRTM). It is very complex topic and understanding
+it could take too much time. What you need to know (or trust us about) is that
+those measurements are done for given piece of code (given piece of system) and
+are stored in **Trusted Platform Module** (TPM) special registers. It gives
+possibility to automatically verify them against corruption and eventually
+proceed or stop further booting. Also, the user has possibility to check if
+measurements are valid. It can be done from OS level using dedicated tools. So,
+as you can see, TrenchBoot is critical piece of firmware, if you want to make it
+secure. Also, its implementation is fully open-source, so everyone can inspect
+it and contribute.
+
+#### Test environment
+
+All requirements of project are automatically or manually validate after
+development. For this purpose, we need to create entire testing infrastructure
+with suitable hardware and software components. Moreover, all tests will be
+published for users, so they can verify development stage on their own
+platforms.
+
+Final results, most likely at the end of each month, will be presented in form
+of blog articles. Everything should be clearly pointed out and explained. All
+used tools, all carried out procedures, all prepared configurations - everything
+user can inspect, revise, build, reproduce and finally use. The goal is to have
+a user-friendly and trust-worthy environment, so community can be sure about the
+quality of the solution we provide.
+
+## Summary
+
+As I mentioned in introduction section, above article is just the beginning of
+the entire series about **DRTM with TrenchBoot implementation**. So stay tuned
+and wait for other articles!
+
+Once again, we would like to thank **NLnet Foundation** for their support. We
+believe that our project will benefit the open-source community and will be a
+big step in development of firmware security.
+
+If you think we can help in improving the security of your firmware or you
+looking for someone who can boost your product by leveraging advanced features
+of used hardware platform, feel free to [book a call with
```

Comment:
Does it render properly with line break here?

Reply:
Revert to original formatting